### DISCUSSION TITLE
Easy Snippet Question

### DISCUSSION THREAD

**Discussion by tjrush** [microsoft/vscode-discussions#473] [Extension Development QnA] [answered]
Is this the proper place to ask questions about a specific extension? If not, can you suggest a proper place? If yes, here's my question.
I'm using the Easy Snippet extension. Is there a way to indicate where the cursor should be after the snippet is loaded?

**Comment by gjsjohnmurray** [comment, marked as answer]
Please ask your question on the repository. It is otften possible to do this by using the Report Issue option from VS Code's Help menu, then choosing "An extension", then the extension.
